R/package.R

```diff
@@ -65,8 +65,8 @@ initialize_python <- function(required_module = NULL, use_environment = NULL) {
   # check for basic python prerequsities
   if (is.null(config)) {
     stop("Installation of Python not found, Python bindings not loaded.")
-  } else if (!is_windows() && (is.null(config$libpython) || !file.exists(config$libpython))) {
-    stop("Python shared library '", config$libpython, "' not found, Python bindings not loaded.")
+  } else if (!is_windows() && is.null(config$libpython)) {
```

Comment:
Can you explain why this change was needed?

Reply:
`config$libpython` is populated by `python_libdir_config()`. Before the proposed change to the function, the library filename is pre-determined. However, this is not the case now. Therefore:
1. I don't think we need to check if the library file exists anymore (`list.files` does this).
2. If no library found, we don't need to print `NULL`.